index.bs

```diff
@@ -2107,6 +2136,11 @@ The [=remote end steps=] with |session| and |command parameters| are:
 
             1. [=list/append=] |partial subscription| to |new subscriptions|.
 
+   1. If |matched events| does not equal |event names|, return [=error=] with [=error code=] [=invalid argument=].
+
+   1. If |top-level traversable context ids| is not empty and |matched contexts| does not equal
```

Comment:
Technically we can simply compare `|matched contexts|` and `|top-level traversable context ids|`, as for global subscriptions they are both empty, for non-global the latest is non-empty. But the current version makes the algo more readable.